OpenAI/Packages/com.openai.unity/Runtime/Completions/CompletionsEndpoint.cs

```diff
@@ -77,11 +77,11 @@ public sealed class CompletionsEndpoint : BaseEndPoint
             int? logProbabilities = null,
             bool? echo = null,
             IEnumerable<string> stopSequences = null,
-            Model model = null,
+            string model = null,
             CancellationToken cancellationToken = default)
         {
             var request = new CompletionRequest(
-                model ?? Model.Davinci,
+                model ?? Model.Davinci : model,
```

Comment:
```suggestion
                Model = string.IsNullOrWhiteSpace(model) ? Model.Davinci : model,
```

Reply:
I think the assignment to Model is wrong here.  (Model = ...)

Reply:
do we need to check again if we do it above?

Reply:
If you mean that we are validating the value of model in both the Endpoint and the Request, yes that is probably redundant.  I would remove the test from the Endpoint and update the parameter comment to remove reference to defaulting.  Up to you.  I think the double testing is probably not a serious perf concern but it is unnecessary.